docs/UserGuide.md

```diff
@@ -15,8 +15,10 @@ Teacher’s Pet can get your contact and class management tasks done faster than
     * [Editing student details: `edit`](#editing-student-details-edit)
     * [Viewing all students: `list`](#viewing-all-students-list)
     * [Finding a student: `find`](#finding-a-student-find)
+    * [Sort displayed students: `sort`](#sort-the-displayed-students-sort)
```

Comment:
would be good to add nested hyperlinks for 
- Sort by name
- Sort by class date
- Sort by money owed

Reply:
The three fields are not broken to go under different subheadings as their contents are very similar. There would not be a hyperlink for these sub-points tho.

Reply:
I have figured out what do you mean by this comment after examining the recent changes and will follow in the upcoming commit.